result/rocev2ipv6.yaml

```diff
@@ -0,0 +1,201 @@
+info:
+  title: RoCEv2 over IPv6results models
+  version: ^0.0.0
+components:
+  schemas:
+    Rocev2IPv6.Metrics.Request:
+      description: >-
+        Request to retrieve RoCEv2 over IPv6 per peer metrics/statistics.
+      type: object
+      properties:
+        choice:
+          type: string
+          description: |- 
+            Fetch stats per peer_name or per QP
+          default: per_peer
+          x-enum:
+            per_peer:
+              x-field-uid: 1
+          x-field-uid: 1
+        per_peer:
+          $ref: '#/components/schemas/Rocev2IPv6.ColumnNames'
+          x-field-uid: 2
+    Rocev2IPv6.ColumnNames:
+      description: >-
+        The names of RoCEv2 over IPv6 peers to return results for.
+        An empty list will return results for all RoCEv2 peers.
+      type: object
+      properties:
+        peer_names:
+          description: >-
+            The names of RoCEv2 over IPv6 peers to return results for.
+            An empty list will return results for all RoCEv2 peers.
+          type: array
+          items:
+            type: string
+          x-constraint:
+          - "/components/schemas/Rocev2.V6peer/properties/name"
+          x-field-uid: 1
+        column_names:
+          description: >-
+            The list of column names that the returned result set will contain.
+            If the list is empty then all columns will be returned except for
+            any result_groups. The name of the RoCEv2 peer cannot be excluded.
+          type: array
+          items:
+            type: string
+            x-enum:
+              session_state:
+                x-field-uid: 1
+              qp_confugred:
+                x-field-uid: 2
+              qp_up:
+                x-field-uid: 3
+              qp_down:
+                x-field-uid: 4
+              connect_request_tx:
+                x-field-uid: 5
+              connect_request_rx:
+                x-field-uid: 6
+              connect_reply_tx:
+                x-field-uid: 7
+              connect_reply_rx:
+                x-field-uid: 8
+              ready_tx:
+                x-field-uid: 9
+              ready_rx:
+                x-field-uid: 10
+              disconnect_request_tx:
+                x-field-uid: 11
+              disconnect_request_rx:
+                x-field-uid: 12
+              disconnect_reply_tx:
+                x-field-uid: 13
+              disconnect_reply_rx:
+                x-field-uid: 14
+              reject_tx:
+                x-field-uid: 15
+              reject_rx:
+                x-field-uid: 16
+              unknown_msg_rx:
+                x-field-uid: 17
+          x-field-uid: 2
+    Rocev2IPv6.Metric.PerPeer:
+      description: >-
+        RoCEv2 per peer statistics information.
+      type: object
+      properties:
+        name:
+          description: >-
+            The name of a configured RoCEv2 peer.
+          type: string
+          x-field-uid: 1
+        session_state:
```

Comment:
I think we should remove this. It is redundant since qp_configured and qp_up/down provides all the data.

Reply:
session_state should only be provided for a specific 'session' . The peer as a session when in fact it contains a group of sessions does not seem right.